### ISSUE TITLE
Backblaze Unable to Find Mounted Harddrives

### ISSUE THREAD

**Issue by amast09** [JonathanTreffler/backblaze-personal-wine-container#60]
I have 4 ext4 hard drives mounted via fstab into my home directory that get mounted into the Backblaze docker container. Unfortunately, Backblaze somehow does not see them.

If I run the wine file browser command I can see all the mounted hard drives and the files within them. I have even tried setting the permissions to 777 on the Linux side of things but Backblaze still doesn't see the mounts.

The fstab mount looks something like this,

```
UUID=3ebc2904-b62e-4303-447c-fg9caf19dbd9 /home/directory-mounted-into-backblaze/Media-1 ext4 user,exec,errors=remount-ro 0 1
```

If there are other details I can provide to help please let me know, thanks for maintaining this repo and any help you can give me!

**Comment by JonathanTreffler** [comment]
Have you tried mounting them as seperate wine devices ?

**Comment by amast09** [comment]
I have not. Unfortunately I have given up attempting to use Backblaze with Linux and am now using a simpler ZFS raidz1 setup instead. I know it isn't offsite, which is a downside, but at this point it is what it is. 😞 

Thanks for replying, I assume when you say `Have you tried mounting them as seperate wine devices` you mean mounting them as their own dedicated docker volumes?

**Comment by JonathanTreffler** [comment]
No, I meant adding d, e, f, g ... wine drives for every directory. So that every harddrive mount on the host has it's own wine drive (not sure if it's own docker volume would help too).

**Comment by amast09** [comment]
If I hadn't reformatted my drive into a ZFS array I would try that. Thank you for your help/replying!

**Comment by Joedon3** [comment]
I'm having problems with the app seeing the data in my zfs volumes.  I found that you can't use any filesystem label as a drive.  You need to create a folder inside the filesystem, then the app will see the data.  In my case, I have sub-filesystems as such:
/zfs/mnt
  /zfs/mnt/d
  /zfs/mnt/n
  /zfs/mnt/p

I tried using the /zfs/mnt folder and it wouldn't backup the data (even though I could browse through it).  I even tried the /zfs/mnt/d and same result.   It only started backing everything up when I mapped /zfs/mnt/d/data/ as drive_d.

I hope that helps....
